cli/failover/failover.go

```diff
@@ -0,0 +1,46 @@
+package failover
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tarantool/cartridge-cli/cli/context"
+	"github.com/tarantool/cartridge-cli/cli/project"
+)
+
+const (
+	defaultFailoverFile = "failover.yml"
+)
+
+// FillCtx function filling the context.Ctx structure
+// for working with the failover
+func FillCtx(ctx *context.Ctx) error {
+	var err error
+
+	if err := project.SetLocalRunningPaths(ctx); err != nil {
+		return err
+	}
+
+	if ctx.Running.AppDir == "" {
+		ctx.Running.AppDir, err = os.Getwd()
+		if err != nil {
+			return fmt.Errorf("Failed to get current directory: %s", err)
+		}
+	}
+
+	if ctx.Running.AppDir, err = filepath.Abs(ctx.Running.AppDir); err != nil {
+		return fmt.Errorf("Failed to get application directory absolute path: %s", err)
+	}
+
+	if ctx.Project.Name == "" {
```

Comment:
This is a copy-paste? Seems that it can be moved to function.

Reply:
Moved to project